site/src/pages/AuditPage/AuditPageView.tsx

```diff
@@ -37,6 +37,27 @@ const presetFilters = [
   { query: "resource_type:user action:delete", name: "Deleted users" },
 ]
 
+const groupAuditLogsByDate = (auditLogs?: AuditLog[]) => {
```

Comment:
if you're willing to use lodash I think this would just be 
```_.groupBy(auditLogs, (log) => new Date(log.time).toDateString())```

Reply:
Yes, I think we could talk about adding lodash since we can import specific functions only instead of the entire package.

Reply:
I think we already have it because it comes bundled with something else